### PR TITLE
Add CLI command to activate votes

### PR DESCRIPTION
**Pull request by asaj**
### Description

This PR adds a CLI command to activate any pending votes for an account, with the option to poll until the epoch is over and the votes can be activated.

### Tested

Manually via against ganache.

### Related issues

- Fixes (partially) #1594 

### Backwards compatibility

Backwards compatible

### CODE REVIEW

File: packages/cli/src/commands/election/activate.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { flags } from '@oclif/command'
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+
+function sleep(ms: number) {
```
Comment (nambrot) [inline packages/cli/src/commands/election/activate.ts:7]: We have a function in @celo/utils/async defined for this

File: packages/cli/src/commands/election/activate.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    const hasPendingVotes = await election.hasPendingVotes(account)
+    if (hasPendingVotes) {
+      if (res.flags.wait) {
+        // Spin until pending votes become activatable.
+        while (!(await election.hasActivatablePendingVotes(account))) {
+          await sleep(1000)
+        }
+      }
+      const txos = await election.activate(account)
+      for (const txo of txos) {
```
Comment (nambrot) [inline packages/cli/src/commands/election/activate.ts:44]: Consider using `concurrentMap` here

File: packages/cli/src/commands/election/revoke.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    await newCheckBuilder(this, res.flags.from)
+      .isSignerOrAccount()
+      .isValidatorGroup(res.flags.for)
+      .runChecks()
+
+    const election = await this.kit.contracts.getElection()
+    const accounts = await this.kit.contracts.getAccounts()
+    const account = await accounts.voteSignerToAccount(res.flags.from)
+    const txos = await election.revoke(account, res.flags.for, new BigNumber(res.flags.value))
+    for (const txo of txos) {
```
Comment (nambrot) [inline packages/cli/src/commands/election/revoke.ts:37]: Ditto `concurrentMap`

File: packages/cli/src/utils/helpers.ts [hunk 4]
```diff
@@ -3,6 +3,7 @@ import { Block } from 'web3/eth/types'
 import { failWith } from './cli'
 
 export async function nodeIsSynced(web3: Web3): Promise<boolean> {
+  return true
```
Comment (nambrot) [inline packages/cli/src/utils/helpers.ts:6]: Is this intended to stay here?

File: packages/contractkit/src/wrappers/Election.ts [hunk 5]
```diff
@@ -100,6 +110,51 @@ export class ElectionWrapper extends BaseWrapper<Election> {
...
+
+  /**
+   * Returns whether or not the account has any pending votes.
+   * @param account The address of the account casting votes.
+   * @return The groups that `account` has voted for.
+   */
+  async hasPendingVotes(account: Address): Promise<boolean> {
+    const groups = await this.contract.methods.getGroupsVotedForByAccount(account).call()
+    const isNotPending = await Promise.all(
+      groups.map(async (g: string) =>
```
Comment (nambrot) [inline packages/contractkit/src/wrappers/Election.ts:141]: Personal preference/Optional Nit: Type `const groups` to avoid having to type the argument here

**Review by nambrot — APPROVED** [review]
Just minor comments

File: packages/cli/src/commands/election/activate.ts (hunk 2 above)
Comment (asaj) [reply packages/cli/src/commands/election/activate.ts:44]: Per @mcortesi's recommendation, intentionally doing this serially.

File: packages/cli/src/commands/election/revoke.ts (hunk 3 above)
Comment (asaj) [reply packages/cli/src/commands/election/revoke.ts:37]: Per @mcortesi's recommendation, intentionally doing this serially.

File: packages/cli/src/utils/helpers.ts (hunk 4 above)
Comment (asaj) [reply packages/cli/src/utils/helpers.ts:6]: Nope.

**Pull request merged**